instance/settings-sample.py

```diff
@@ -62,3 +62,11 @@
 SUPPORT_EMAIL = 'person@example.com'
 #: Sitemap key
 SITEMAP_KEY = None
+DOGPILE_CACHE_URLS = "http://127.0.0.1:6379"
+DOGPILE_CACHE_REGIONS = [
+    ('hour', 3600),
+    ('hasjob_index', 3600),
+    ('day', 3600 * 24),
+    ('month', 3600 * 24 * 31),
```

Comment:
Remove hour/day/month. They're not needed. Put all these Dogpile settings in `settings.py` as well because Hasjob will not run without these specific values.